docs/sources/datasources/elasticsearch.md

```diff
@@ -75,6 +75,12 @@ number followed by a valid time identifier, e.g. `1m` (1 minute) or `30s` (30 se
 | `s`        | second      |
 | `ms`       | millisecond |
 
+### Piped processing language (PPL) support
+
+PPL support can be enabled in the datasource configuration setting when Elasticsearch version is set to 7.0+. See [Open Distro for Elasticsearch SQL plugin](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#sql) for plugin installation guide and [plugin compatibility](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#plugin-compatibility) for version compatibility information.
```

Comment:
do we want to call that PPL will available after opendistro_sql 1.11.0.0

Reply:
Yep. Will change to specify the version of plugin.